pkgs/by-name/ma/mattermost/update.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts prefetch-npm-deps
+set -euo pipefail
+
+# Get latest ESR version (9.11.x series)
+version=$(curl ${GITHUB_TOKEN:+" -u \":$GITHUB_TOKEN\""} -s https://api.github.com/repos/mattermost/mattermost/releases | \
+    jq -r '[.[] | select(.tag_name | test("^v9\\.11\\.[0-9]+$"))] | .[0].tag_name | sub("^v"; "")')
```

Comment:
Probably makes sense to have this test argument (the version regex) in a bash variable toward the top and pass it into jq with --arg. No double escaping necessary.